vendor/ember-mocha/test-loader.js

```diff
@@ -1,31 +1,41 @@
-/* globals jQuery, mocha, require, requirejs */
+/* globals mocha, require, requirejs */
+(function() {
 
-jQuery(document).ready(function() {
-  var testLoaderModulePath = 'ember-cli-test-loader/test-support/index';
-
-  if (!requirejs.entries[testLoaderModulePath]) {
-    testLoaderModulePath = 'ember-cli/test-loader';
+  function ready(fn) {
+    if (document.readyState !== 'loading') {
+      fn();
+    } else {
+      document.addEventListener('DOMContentLoaded', fn);
+    }
   }
 
-  var TestLoader = require(testLoaderModulePath)['default'];
-  TestLoader.prototype.shouldLoadModule = function(moduleName) {
-    return moduleName.match(/[-_]test$/) || moduleName.match(/\.jshint$/);
-  };
+  ready(function() {
+    var testLoaderModulePath = 'ember-cli-test-loader/test-support/index';
+
+    if (!requirejs.entries[testLoaderModulePath]) {
+      testLoaderModulePath = 'ember-cli/test-loader';
+    }
+
+    var TestLoader = require(testLoaderModulePath)['default'];
+    TestLoader.prototype.shouldLoadModule = function(moduleName) {
+      return moduleName.match(/[-_]test$/) || moduleName.match(/\.jshint$/);
+    };
 
-  TestLoader.prototype.moduleLoadFailure = function(moduleName, error) {
-    describe('TestLoader Failures', function () {
-      it(moduleName + ': could not be loaded', function() {
-        throw error;
+    TestLoader.prototype.moduleLoadFailure = function(moduleName, error) {
+      describe('TestLoader Failures', function() {
+        it(moduleName + ': could not be loaded', function() {
+          throw error;
+        });
       });
-    });
-  };
+    };
 
-  // Attempt to mitigate sourcemap issues in Chrome
-  // See: https://github.com/ember-cli/ember-cli/issues/3098
-  //      https://github.com/ember-cli/ember-cli-qunit/pull/39
-  setTimeout(function() {
-    TestLoader.load();
+    // Attempt to mitigate sourcemap issues in Chrome
+    // See: https://github.com/ember-cli/ember-cli/issues/3098
+    //      https://github.com/ember-cli/ember-cli-qunit/pull/39
+    setTimeout(function() {
+      TestLoader.load();
 
-    window.mochaRunner = mocha.run();
-  }, 250);
-});
+      window.mochaRunner = mocha.run();
```

Comment:
@Turbo87 Unrelated to the changes here, but do we actually need this assignment? 

This was actually annoying when tracking down memory leak issues in an app recently, as it was leaking stuff itself. So when running tests to automate some application flow and checking if e.g. the container is GCed correctly afterwards, this effectively prevented this approach as you could never be sure why something was leaking. Removing the assignment was enough to remove the leak.

So happy to remove that, if there is no (*very!*) good reason to have it! ;)

Reply:
Might even fix https://github.com/emberjs/ember-mocha/pull/168?

Reply:
to be honest, I really don't know. my initial thought was that it might be required by testem? but if it works fine for you without it then I don't see a reason to keep it.

maybe @rwjblue has an idea?

Reply:
Yes, I just patched that file quickly in my `node_modules`, and everything was running fine as before!